### DISCUSSION TITLE
Can the intelligent prompt for pathlib on Jupiter be applied to Python files

### DISCUSSION THREAD

**Discussion by ydmykr** [microsoft/pylance-release#3133] [Enhancement]
As the title said, when I was using jupyter, I was pleasantly surprised to find that there was a prompt when using the code:

```
from pathlib import Path
cwd=Path.cwd()

cwd / ""
```
![image](https://user-images.githubusercontent.com/100815370/182559541-6bc225b1-c089-47eb-a934-b4f16c4d6b2e.png)

I tried in the python file, but I didn't get a similar prompt. I hope the python extension can also support this.

![image](https://user-images.githubusercontent.com/100815370/182558453-c8d86f8a-74d4-49ec-b1da-d8ed0464cfb4.png)

**Comment by YDX-2147483647** [comment]
I guess it's theoretically impossible. The working directory is uncertain until we run it.
